### DISCUSSION TITLE
Pony Island

### DISCUSSION THREAD

**Discussion by Leilei332** [Gamuxorg/bbs#282] [投递和请求游戏]
游戏名称：小马岛
游戏名称(英文)：Pony Island
下载地址 :（如果有的话）：https://rutracker.org/forum/viewtopic.php?t=5180929

这是我自己上传到网盘里的：https://www.jianguoyun.com/p/DRBbUo8QnNWwCRiAr4ME

**Comment by Leilei332** [comment]
Steam:https://store.steampowered.com/app/405640/Pony_Island/

官网:https://www.pony-island.com

**Comment by hyh19962008** [comment]
你好，如果已经打包好了的话，可以尝试使用站内的投稿功能，在任意游戏文章页面，下载按钮旁边有一个"投稿"的按钮
